Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -11,6 +11,8 @@
 #include <bimg/decode.h>
 #include <bimg/encode.h>
 
+#define STB_IMAGE_RESIZE_IMPLEMENTATION
```

Comment:
If we add a CMake target as described in my other comment, this could be defined in that target so that we don't have to worry about it here.

Reply:
What's annoying with STB include file is only 1 .cpp file in the solution must define `STB_IMAGE_RESIZE_IMPLEMENTATION` or any other include will popup link error. I prefer to let this define clearly visible just before the #include statement.

Reply:
I see. Can you note this nuance in a comment just above the define? That'll just help future developers know that we usually handle compiler defines differently and that this one is the exception rather than the rule.